ble/UUID.h

```diff
@@ -42,7 +97,7 @@ class UUID {
      *         different service or characteristics on the BLE device.
      *
      * @param longUUID
-     *          The 128-bit (16-byte) UUID value, MSB first (big-endian).
+     *          The 128-bit (16-byte) UUID value, LSB first (little-endian).
```

Comment:
I also feel that we should add a document header before `getBaseUUID` stating that it will fetch an internal representation of the UUID bits in little-endian form.